apps/backend/src/schemata/schema.ts

```diff
@@ -148,3 +148,20 @@ export const unpublishBodySchema = object({
 
 export const parseUnpublishBody = (input: unknown) =>
   parse(unpublishBodySchema, input);
+
+export const codemodRunBodySchema = object({
+  source: string(),
+  codemodName: string(),
+  engine: union([literal("jscodeshift"), literal("ts-morph")]),
```

Comment:
Why we are limiting engine to only jscodeshift and ts-morph? Other engines dont not support "run from source"?

Reply:
@DmytroHryshyn i've picked them from [here](https://linear.app/codemod/issue/CDMD-3089/%5Bbackend%5D-add-endpoint-to-run-the-codemod)